lib/src/solver/result.dart

```diff
@@ -109,6 +109,7 @@ class SolveResult {
         SolveReport(type, _sources, _root, _previousLockFile, this, cache);
     report.summarize(dryRun: dryRun);
     if (type == SolveType.UPGRADE) {
+      await report.reportDiscontinued();
```

Comment:
Hmm - question is if we only want to do this on `upgrade` or also on `get`? I think we would like to show this always...

@jonasfj WDYT?

Reply:
We decided for now this is good!